fe/fe-core/src/main/java/com/starrocks/qe/scheduler/QueryRuntimeProfile.java

```diff
@@ -304,15 +304,17 @@ public void updateProfile(FragmentInstanceExecState execState, TReportExecStatus
         long lastTime = lastRuntimeProfileUpdateTime.get();
         Supplier<RuntimeProfile> topProfileSupplier = this.topProfileSupplier;
         ExecPlan plan = execPlan;
-        if (topProfileSupplier != null && plan != null && connectContext != null &&
-                connectContext.isProfileEnabled() &&
+        if (topProfileSupplier != null && plan != null && (connectContext != null &&
+                // broker load is async job, we can't get the job running time through the session start time,
```

Comment:
why can't get the job running time from the session? Do you mean can't get the  time if job is replayed? If so, we can set the start time with `LoadJob.createTimestamp`?

Reply:
we can submit many broker load job during one session.